### PR TITLE
Release 202.0.0

### PR DESCRIPTION
**Pull request by OGPoyraz**
## Explanation

This PR is release candidate for v202.0.0

### Changes

- Bumped `@metamask/controller-utils` to `^11.3.0` for following packages
    - `@metamask/address-book-controller`
    - `@metamask/assets-controller`
    - `@metamask/ens-controller`
    - `@metamask/gas-fee-controller`
    - `@metamask/logging-controller`
    - `@metamask/message-manager`
    - `@metamask/name-controller`
    - `@metamask/network-controller`
    - `@metamask/notification-services-controller`
    - `@metamask/permissions-controller`
    - `@metamask/polling-controller`
    - `@metamask/preferences-controller`
    - `@metamask/polling-controller`
    - `@metamask/queued-request-controller`
    - `@metamask/user-operation-controller`
    - `@metamask/transaction-controller`
    - `@metamask/signature-controller`

- Changes for `@metamask/transaction-controller`
    - Removed trace types from `TransactionController`, instead import them from `@metamask/controller-utils` ([#4655](https://github.com/MetaMask/core/pull/4655))
- Changes for `@metamask/signature-controller`
    - Add tracing infrastructure ([#4575](https://github.com/MetaMask/core/pull/4575))
    - Add optional `trace` callback to constructor.
    - Add optional `traceContext` option to `newUnsignedPersonalMessage` and `newUnsignedTypedMessage` methods.
    - Add initial tracing of signature lifecycle.

## Checklist

- [x] I've updated the test suite for new or updated code as appropriate
- [x] I've updated documentation (JSDoc, Markdown, etc.) for new or updated code as appropriate
- [x] I've highlighted breaking changes using the "BREAKING" category above as appropriate

### CODE REVIEW

File: packages/signature-controller/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
+## [19.1.0]
+
+### Changed
+
+- Add tracing infrastructure ([#4575](https://github.com/MetaMask/core/pull/4575))
+  - Add optional `trace` callback to constructor.
+  - Add optional `traceContext` option to `newUnsignedPersonalMessage` and `newUnsignedTypedMessage` methods.
+  - Add initial tracing of signature lifecycle.
+
```
Comment (MajorLift) [inline packages/signature-controller/CHANGELOG.md:18]:
```suggestion
- Bump `@metamask/controller-utils` from `^11.2.0` to `^11.3.0` ([#4688](https://github.com/MetaMask/core/pull/4688))

```

File: packages/transaction-controller/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [36.2.0]
+
+### Changed
+
+- Removed trace types from `TransactionController`, instead import them from `@metamask/controller-utils` ([#4655](https://github.com/MetaMask/core/pull/4655))
+
```
Comment (MajorLift) [inline packages/transaction-controller/CHANGELOG.md:15]:
```suggestion
- Bump `@metamask/controller-utils` from `^11.2.0` to `^11.3.0` ([#4688](https://github.com/MetaMask/core/pull/4688))

```

**Review by MajorLift — COMMENTED** [review]
Added entries for `controller-utils` bumps.

It would also be nice to note the `controller-utils` bump in the PR description. It would be very helpful to future changelog writers releasing packages that updated their `controller-utils` versions in this pr but were not part of this release.

**Comment by MajorLift** [comment]
Looks like the lockfile needs an update. Otherwise LGTM!

**Comment by OGPoyraz** [comment]
Thanks a lot for your input @MajorLift

File: packages/signature-controller/CHANGELOG.md (hunk 1 above)
Comment (MajorLift) [reply packages/signature-controller/CHANGELOG.md:18]: @OGPoyraz Looks like these entries got overwritten

File: packages/signature-controller/CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [19.1.0]
+
+### Changed
```
Comment (MajorLift) [inline packages/signature-controller/CHANGELOG.md:12]: Would "Added" be a better category for these changes?

```suggestion
### Added
```

File: packages/transaction-controller/CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [36.2.0]
+
```
Comment (MajorLift) [inline packages/transaction-controller/CHANGELOG.md:11]:
```suggestion

### Changed

- Bump `@metamask/controller-utils` from `^11.2.0` to `^11.3.0` ([#4688](https://github.com/MetaMask/core/pull/4688))

```

File: packages/transaction-controller/CHANGELOG.md (hunk 2 above)
Comment (MajorLift) [reply packages/transaction-controller/CHANGELOG.md:15]: Replaced with https://github.com/MetaMask/core/pull/4688/files#r1758413732

File: packages/signature-controller/CHANGELOG.md (hunk 1 above)
Comment (MajorLift) [reply packages/signature-controller/CHANGELOG.md:18]: Updated suggestion to reflect category change.

**Comment by MajorLift** [comment]
The controller-utils bump entry in the signature-controller changelog also needs to be restored. You might need to refresh the page to see the updates on [this thread](https://github.com/MetaMask/core/pull/4688#discussion_r1756725455) above.

File: packages/transaction-controller/CHANGELOG.md [hunk 5]
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### Changed
+
+- Bump `@metamask/controller-utils` from `^11.2.0` to `^11.3.0` ([#4688](https://github.com/MetaMask/core/pull/4688))
+
+### Removed
+
+- Removed trace types from `TransactionController`, instead import them from `@metamask/controller-utils` ([#4655](https://github.com/MetaMask/core/pull/4655))
```
Comment (matthewwalsh0) [inline packages/transaction-controller/CHANGELOG.md:18]: Should we highlight this as a breaking change?

**Review by matthewwalsh0 — CHANGES_REQUESTED** [review]

File: packages/transaction-controller/CHANGELOG.md (hunk 5 above)
Comment (MajorLift) [reply packages/transaction-controller/CHANGELOG.md:18]: We might also want to enumerate the affected type names:

```suggestion
- **BREAKING:** Removed types `TraceContext`, `TraceRequest`, `TraceCallback` from `TransactionController` ([#4655](https://github.com/MetaMask/core/pull/4655))
  - These types have been moved to `@metamask/controller-utils`.
```

Comment (OGPoyraz) [reply packages/transaction-controller/CHANGELOG.md:18]: Good point but this is not a breaking change, those types never exported from `TransactionController`, just used internally.

File: packages/signature-controller/CHANGELOG.md (hunk 1 above)
Comment (OGPoyraz) [reply packages/signature-controller/CHANGELOG.md:18]: Done in https://github.com/MetaMask/core/pull/4688/commits/104bf0f49683ac17a730979ba6ef188ebaf729eb

File: packages/transaction-controller/CHANGELOG.md (hunk 5 above)
Comment (MajorLift) [reply packages/transaction-controller/CHANGELOG.md:18]: Ah I see that they're not listed in `index.ts`. In that case, we could also remove this entry altogether, although leaving it also works if its relevant to the team.

**Review by MajorLift — APPROVED** [review]

File: packages/transaction-controller/CHANGELOG.md [hunk 6]
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [36.2.0]
+
+### Changed
+
+- Bump `@metamask/controller-utils` from `^11.2.0` to `^11.3.0` ([#4688](https://github.com/MetaMask/core/pull/4688))
```
Comment (matthewwalsh0) [inline packages/transaction-controller/CHANGELOG.md:14]: Apologies for the hassle, but we'll need to add the change log entries for the startup fixes also, thanks!

https://github.com/MetaMask/core/pull/4658

**Review by matthewwalsh0 — CHANGES_REQUESTED** [review]

**Review by Mrtenz — CHANGES_REQUESTED** [review]
We should do a release of all packages now that #4648 is merged. Happy to recreate this PR for you.

**Comment by OGPoyraz** [comment]
We decided to close this and create new one with @Mrtenz

**Pull request closed without merging**